release_notes.md

```diff
@@ -1,118 +1,57 @@
-    - `Outbreak_Alert-Time-Frame-Analysis`: Efficiently identify and deactivate outdated outbreak alerts based on their investigation time frame.
-    - `Outbreak_Automated-Deployment-Outbreak-Alert-Response-Solution-Pack`: Seamlessly install the latest outbreak alert solutions pack (from the past X days) with automated email notifications.
-    - `Outbreak_Ingest-Tracking-Outbreak-CVEs-and-IOCs`: Retrieve outbreak alert CVEs (KVE) from NIST and IOCs from Fortinet FortiGuard, focusing on those marked as ‘Tracking’.
+### NIST NVD Connector Action Isolation
 
-- As an investigation result, we now create incidents in place of alerts.
+The **NIST NVD Connector** action, responsible for fetching CVE details, has been isolated into a dedicated playbook named **Update CVE Details from NIST**. This playbook includes the setting `Ignore Error` set to *Yes* to handle intermittent API issues more gracefully.  
 
-#### Pluggable Threat Hunt Framework
+The **Update CVE Details from NIST** playbook is referenced as the final step in the **Find Known Exploited Vulnerabilities (KEV) CVEs** playbook. This change addresses the intermittent reliability issues with the NIST NVD APIs and ensures uninterrupted processing of CVE data.
 
-The Pluggable Threat Hunting framework integrates threat detection tools, such as FortiSIEM, FortiAnalyzer, QRadar, and Splunk, for efficient outbreak alert threat hunting and integrates the following hunting categories:
+### Key Store Enhancements
 
-- Fabric Hunting
-- Indicator of Compromise (IoC) Hunting
-- Signature Hunting
+- Key Store records associated with Outbreaks are now tagged with **`Outbreak Alert`**
 
-#### Widget Additions
+### Schedule Changes for Solution Pack Deployment
 
-We’ve integrated cutting-edge widgets to execute your playbooks with a single click and track the execution with an intuitive, user-friendly wizard.
+The crontab schedule for **Outbreak_Automated-Deployment-Outbreak-Alert-Response-Solution-Pack** has been adjusted to run **every 3 hours**. This ensures the timely deployment of new outbreak alert solution packs.  
 
-- Playbook Execution Wizard
-- Playbook Buttons
+These solution packs, created upon receiving outbreak alerts from FortiGuard, include YARA rules, Sigma rules, and other detection mechanisms designed to facilitate the hunting and identification of outbreaks within a network.
 
-#### Picklist Enhancements
+### Playbook Enhancements and Cleanups
 
-The following picklists and the containing items were renamed to provide clearer context and more specific terminology, reflecting its focus on outbreak-related alerts. This change enhances user understanding and aligns with the outbreak management system's terminology, making it easier for analysts to identify and manage relevant outbreak alerts.
+- The following steps have been removed from the **Investigate Outbreak** playbook:
```

Comment:
Please reword this point as Optimised **Investigate Outbreak** playbook by removing duplicate steps. No need to mention about which steps are removed as user is not interested in steps but the functioanality.

Reply:
Done